### PR TITLE
added beta parameter to cemitool

### PR DESCRIPTION
**Pull request by hechth**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [x] This PR updates an existing tool or tool collection

### CODE REVIEW

**Comment by hechth** [comment]
@nomadscientist @nilchia please check if this works for you.

File: tools/cemitool/cemitool.xml [hunk 1]
```diff
@@ -103,6 +104,8 @@
                 </sanitizer>
                 <validator type="regex">[0-9a-zA-Z:-_]+</validator>
             </param>
+            <param name="beta" type="integer" min="0" value="2" label="Beta value"
+                help="The beta value is a parameter that lies in the core of the weighted gene co-expression network analysis (WGCNA)."/>
```
Comment (nilchia) [inline tools/cemitool/cemitool.xml:108]: The default value is NULL.

**Comment by nilchia** [comment]
It worked for me locally. Thanks @hechth

File: tools/cemitool/cemitool.xml (hunk 1 above)
Comment (hechth) [reply tools/cemitool/cemitool.xml:108]: So the parameter is optional?

**Review by SaimMomin12 — APPROVED** [review]

**Comment by SaimMomin12** [comment]
Fixes: https://github.com/galaxyproject/tools-iuc/issues/6246

File: tools/cemitool/cemitool.xml (hunk 1 above)
Comment (nilchia) [reply tools/cemitool/cemitool.xml]: Yes, by default it calculates the beta value by soft thresholding and assigns it automatically.

**Review by nilchia — APPROVED** [review]
Thanks @hechth

File: tools/cemitool/cemitool.xml [hunk 2]
```diff
@@ -17,6 +17,9 @@
         #if $interactions
             -I '$interactions'
         #end if
+        #if $beta
```
Comment (bgruening) [inline tools/cemitool/cemitool.xml:20]:
```suggestion
        #if str($beta):
```

Comment (bgruening) [reply tools/cemitool/cemitool.xml:20]: in the case the integer is 0

**Review by Mennayousef — APPROVED** [review]

**Comment by nomadscientist** [comment]
> It worked for me locally. Thanks @hechth

Thank you so much for fixing & for testing! This really helps this internship student out (they weren't set up to run a local instance)

**Comment by nomadscientist** [comment]
This will take a week to get picked up / updated on the big 3, right?

**Comment by bgruening** [comment]
I can update it on EU, now if you like.

**Comment by nomadscientist** [comment]
Oh yes please!

**Pull request merged**